docs/source/getting_started/woodwork_types.ipynb

```diff
@@ -0,0 +1,386 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aa1966fd",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = es['products']\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "164b1138",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.ww"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4bffac54",
+   "metadata": {},
+   "source": [
+    "Notice how the three columns showing this DataFrame's typing information are the three elements of typing information outlined at the beginning of this guide. To reiterate: By defining physcial types, logical types, and semantic tags for each column in a DataFrame, we've defined a DataFrame's Woodwork schema, and with it, we can gain an understanding of the contents of each column.\n",
+    "\n",
+    "This column-specific typing information that exists for every column in every DataFrame in an EntitySet is an integral part of Deep Feature Synthesis' ability to generate features for an EntitySet.\n",
+    "\n",
+    "### Woodwork in DFS\n",
+    "As the units of computation in Featuretools, Primitives need to be able to specify the input types that they allow as well as have a predictable return type. For an in-depth explanation of Primitives in Featuretools, see the [Feature Primitives](primitives.ipynb) guide. Here, we'll look at how the Woodwork types come together into a `ColumnSchema` object to describe Primitive input and return types.\n",
+    "\n",
+    "Below is a Woodwork `ColumnSchema` that we've obtained from the `'product_id'` column in the `products` DataFrame in the retail EntitySet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "349e5274",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "products_df = es['products']\n",
+    "product_ids_series = products_df.ww['product_id']\n",
+    "column_schema = product_ids_series.ww.schema\n",
+    "column_schema"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8e8c0ccf",
+   "metadata": {},
+   "source": [
+    "This combination of logical type and semantic tag typing information is a `ColumnSchema`. In the case above, the `ColumnSchema` describes the **type definition** for a single column of data. \n",
+    "\n",
+    "Notice that there is no physical type in a `ColumnSchema`. This is because a `ColumnSchema` is a collection of Woodwork types that doesn't have any data tied to it and therefore has no physical representation. Because a `ColumnSchema` object is not tied to any data, it can also be used to describe a **type space** into which other columns may or may not fall.\n",
+    "\n",
+    "This flexibility of the `ColumnSchema` class means that each column in an EntitySet ultimately has its own type definition `ColumnSchema`, and each Primitive in Featuretools defines the input and return types using a `ColumnSchema` type space.\n",
```

Comment:
The clause after the comma sounds a bit awkward, especially if we remove the preceding statement

> This flexibility of the `ColumnSchema` class means that each Primitive in Featuretools defines the input and return types using a `ColumnSchema` type space.

How about 

> This flexibility of the `ColumnSchema` class means that each column in an EntitySet ultimately has its own type definition `ColumnSchema`.  Each Primitive in Featuretools defines the input and return types using a `ColumnSchema` type space. 

or adding a different verb after the comma

Reply:
Yeah, I see what you're saying about the awkwardness here. The point I'm trying to hammer in here is that the flexibility of `ColumnSchemas` allows them to be used for these two purposes, and I think that the original form of this sentence isn't getting that point across so well anyway. I'm thinking of making this change:

> This flexibility of the `ColumnSchema` class allows `ColumnSchema` objects to be used both as type definitions for every column in an EntitySet as well as input and return type spaces for every Primitive in Featuretools.

Reply:
Yeah, that change sounds great!